src/main/java/org/owasp/wrongsecrets/MvcConfiguration.java

```diff
@@ -10,9 +10,12 @@
 import org.springframework.context.ApplicationContext;
 import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Configuration;
+import org.springframework.web.servlet.config.annotation.EnableWebMvc;
```

Comment:
:wave: 

This import is not used, could you please remove it? :slightly_smiling_face:

Reply:
Removed :-)